nodes/vector/turbulence.py

```diff
@@ -148,18 +148,15 @@ def get_offset(seed):
                 offset = noise.random_unit_vector() * 10.0
             return offset
 
-        for idx, (args) in enumerate(zip(verts, m_octaves, m_hard, m_amp, m_freq, m_seed)):
-            print('from args[0]: ',args[0])
-            seed = args[5]
-            vert_list = args[0]
-
-            if not seed == 0.0:
-                offset = get_offset(seed)
-                verts = [[v[0] + offset[0], v[1] + offset[1], v[2] + offset[2]] for v in vert_list]
-
-            _noise_type = noise_dict[self.noise_type]
-            octaves, hard, amp, freq = args[1:5]
-            out.append([turbulence_f[self.out_mode](v, octaves, hard, _noise_type, amp, freq) for v in verts])
+            arguments = verts, m_octaves, m_hard, m_amp, m_freq, m_seed
+            for idx, (vert_list, octaves, hard, amp, freq, seed) in enumerate(zip(*arguments)):
+
+                if not seed == 0.0:
+                    offset = get_offset(seed)
+                    verts = [[v[0] + offset[0], v[1] + offset[1], v[2] + offset[2]] for v in vert_list]
```

Comment:
don't reuse the variable name.. verts... unless you call the origin verts `m_verts`.